pkg/cluster/install.go

```diff
@@ -280,6 +280,8 @@ func (m *manager) bootstrap() []steps.Step {
 		steps.Action(m.populateMTUSize),
 
 		steps.Action(m.createDNS),
+		// TODO: Add a condition on ServicePrincipalProfile to run createOIDC after testing
+		steps.Action(m.createOIDC),
```

Comment:
Are we allowing customers to update their clusters to consume OIDC / workload identities that the customer can specify?  IF so, are we doing that here?  

`az aro update --enable-oidc` something like that.

Reply:
I don't think we have the scope around customer providing the OIDC IssuerURL, it will always be managed and set by RP.
@cadenmarchese can confirm.

Reply:
It might be in-scope eventually, but for now I think we can table it.